src/concerns/dimensions.coffee

```diff
@@ -7,14 +7,20 @@ export default
 		viewportWidth: null # Width of the viewport, for media query calculation
 		carouselWidth: null # Width of a page of the carousel
 		gutterWidth: 0 # Computed width of gutters, since they support css vars
+		resizeObserver: null
```

Comment:
We can ditch this because it isn't reactive state

```suggestion
```

Reply:
👍